src/include/nanobench.h

```diff
@@ -3266,6 +3268,11 @@ std::vector<BigO> Bench::complexityBigO() const {
     return bigOs;
 }
 
+void Bench::clearResults() {
+    mResults.clear();
+    mConfig.mContext.clear();
+}
+
```

Comment:
Is this a good or a bad idea? I'm happy to rebase and throw it out again.

Reply:
Alternatively, one could add `Bench::clearContext()`, but I assume if the benchmark title changes the set of context variables will change as well. Therefore I'd say it is better to clear the context as well.

Reply:
> Given that there is a constructor std::string(char const*), this conversion should happen implicitly

I originally only had std::string arguments, then added the `char const*` ones because using them compiles a bit faster. That's probably not really relevant though. Ideally I should use `std::string_view` everywhere but that doesn't exist in C++11.

The methods could probably cleaned up a bit, I didn't find it worth the effort though.

> Is this a good or a bad idea?

I think its better to not clean the context here.